client-misk-dynamodb/build.gradle.kts

```diff
@@ -21,6 +21,7 @@ dependencies {
   testApi(project(":client-misk-testing"))
 
   implementation(Dependencies.misk)
+  implementation(Dependencies.miskAwsDynamodb)
```

Comment:
I think this dependency only needs to be a testImplementation

Reply:
Didn't mean to approve

Reply:
Will fix